frontend/src/asyncs/moderation.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  MODERATION_APPS_URL,
+  MODERATION_APP_URL,
+  MODERATION_REVIEW_URL,
+} from "src/env"
+import { ModerationApp, ModerationApps } from "src/types/Moderation"
+
+/**
+ * Performs API request to retrieve list of apps for moderation
+ * @returns array of apps
+ */
+export async function getModerationApps(): Promise<ModerationApps> {
+  let res: Response
+  try {
+    res = await fetch(MODERATION_APPS_URL, { credentials: "include" })
```

Comment:
Just FYI, I might start replaceing these asyncs with something like https://tanstack.com/query/latest/docs/react/quick-start in the near future